### PR TITLE
rebuild MPS_AKH to include HOU innovations

### PR DESCRIPTION
**Pull request by gwax**

### CODE REVIEW

File: web/changelog.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
 [
   {
+    "version": "3.8.x",
```
Comment (fenhl) [inline web/changelog.json]: This should probably be merged into a 3.8.7 release branch along with #383.

**Comment by fenhl** [comment]
Travis error seems unrelated.

File: web/changelog.json (hunk 1 above)
Comment (gwax) [inline web/changelog.json]: I figure that is best left as the responsibility of whoever cuts the actual release (as well as adding the date).

**Comment by gwax** [comment]
Travis has been having this specific error a lot recently. I want to dig into it at some point but haven't had a chance.

File: web/changelog.json (hunk 1 above)
Comment (gwax) [inline web/changelog.json]: What if I call it `master` instead?

**Comment by fenhl** [comment]
Rebase onto future/3.8.7 branch please.

**Comment by gwax** [comment]
rebase complete

File: web/changelog.json [hunk 2]
```diff
@@ -1,5 +1,17 @@
 [
   {
+    "version": "master",
```
Comment (ZeldaZach) [inline web/changelog.json]: Should really put the next version number here

**Pull request merged**